### PR TITLE
Wiki 8 Private Wikis: complete

### PR DESCRIPTION
**Pull request by estenman**
The functionality works but I did notice I need to work on setting up sessions.

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -46,7 +46,7 @@
 <% if current_user.role == "standard" %>
   <%= link_to "Upgrade to Premium Now", new_charge_path %>
 <% elsif current_user.role == "premium" %>
-  <%= link_to "Downgrade to standard", user.downgrade_role  %>
+  <%= link_to "Downgrade to standard", current_user.downgrade_role  %>
```
Comment (christine-le) [inline app/views/devise/registrations/edit.html.erb]: This currently results in an error, when we click the link.  We should link to a __path__ instead (that should be defined in the Charges controller to downgrade the user)

File: config/environments/production.rb [hunk 2]
```diff
@@ -78,6 +78,6 @@
   config.active_record.dump_schema_after_migration = false
 
   #Devise default urls for action mailer options
-  config.action_mailer.default_url_options = { host: 'intense-escarpment-81403.herokuapp.com' }
+  config.action_mailer.default_url_options = { host: 'intense-escarpment-81403.herokuapp.com', port: 3000 }
```
Comment (christine-le) [inline config/environments/production.rb:81]: Nice to see this site deployed to Heroku!  You may need to run some commands to remotely create or migrate the db.  I ran into some errors when I tried to login.  This is low priority though, I can show you in our next meeting how to do this...

**Comment by estenman** [comment]
All fixed! It took me a little while to figure out that I needed to create another attribute to hold the stripe customer ID but I did figure it out. :)

**Comment by christine-le** [comment]
Nice job!

**Comment by christine-le** [comment]
Can you give me "write access" so I can approve / merge the PR?  Thanks!

**Pull request merged**